src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -403,3 +403,85 @@ document.body.addEventListener(
+
+## Reducing Bundle Size
+
+Sentry ships with code that enables you to <PlatformLink to="/configuration/options/#debug">debug your Sentry configuration</PlatformLink>. While this code can be very useful in development environments, it's not typically necessary to include it in your production bundles where it takes up valuable space. The JavaScript SDK uses special flags in its CommonJS and ESM distributions to facilitate tree shaking of code sections that are used for debugging. Tree shaking is a way to remove unused code in your application during the build process.
+
+<PlatformSection notSupported={["javascript.nextjs"]}>
+To mark any debug code as unused, you must replace debug flags in the Sentry SDK with `false`. We've outlined examples of how to do this in popular toolchains below.
+</PlatformSection>
+
+<PlatformSection notSupported={["javascript.nextjs"]}>
+
+### Tree Shaking Debug Code With webpack
+
+To tree shake Sentry debug code in your webpack bundles, we recommend using webpack's [DefinePlugin](https://webpack.js.org/plugins/define-plugin/). Use it to replace all occurrences of the `__SENTRY_DEBUG__` flag with `false`:
+
+```javascript {filename:webpack.config.js}
+const webpack = require("webpack");
+
+module.exports = {
+  // ... other options
+  plugins: [
+    new webpack.DefinePlugin({
+      __SENTRY_DEBUG__: false,
+    }),
+    // ... other plugins
+  ],
+};
+```
+
+</PlatformSection>
+
+<PlatformSection notSupported={["javascript.nextjs"]}>
+
+### Tree Shaking Debug Code With rollup.js
+
+If you're using rollup.js, we recommend [rollup's replace plugin](https://www.npmjs.com/package/@rollup/plugin-replace) to replace all occurrences of `__SENTRY_DEBUG__` with `false`:
+
+```javascript {filename:rollup.config.js}
+import replace from "@rollup/plugin-replace";
+import { terser } from "rollup-plugin-terser";
+
+export default {
+  // ... other options
+  treeshake: "smallest", // recommended for best tree shaking results
+  plugins: [
+    replace({
+      preventAssignment: false, // not necessary because flags are not assigned to
+      values: {
+        __SENTRY_DEBUG__: false,
+      },
+    }),
+    // ... other plugins (best placed after)
+  ],
+};
+```
+
+</PlatformSection>
+
+<PlatformSection supported={["javascript.nextjs"]}>
+
+### Tree Shaking Debug Code With Next.js
+
+To tree shake Sentry debug code in Next.js projects, you can use webpack's [DefinePlugin](https://webpack.js.org/plugins/define-plugin/). The following Next.js configuration will replace all occurrences of `__SENTRY_DEBUG__` in your bundles with `false`:
+
+```javascript {filename:next.config.js}
+const nextConfig = {
+  webpack: (config, { webpack }) => {
```

Comment:
You've tested this, I assume? Is the second `webpack` here actually the webpack module? (The next docs don't say.)

Reply:
Yeah it's actually the webpack module. (Questions like these always make me double-doubt my process but I'm gonna trust it here.)